specification/src/main/asciidoc/platform/Security.adoc

```diff
@@ -874,8 +874,7 @@ A Jakarta EE Product Provider may choose to enable
 selective access to resources using the Java protection model. The
 mechanism used is Jakarta EE product dependent.
 
-The _permissions.xml_ descriptor (see
-<<a2496, Declaring Permissions Required by Application Components>>) makes it possible to express
+The _permissions.xml_ descriptor makes it possible to express
```

Comment:
Depending on the discussion in the mailing list this entire sentence may be removed.  I do wonder if the entire `Code Authorization` section should be removed.

Reply:
I still wonder if the whole `Code Authorization` section should be removed.